src/controllers/expenseController.js

```diff
@@ -0,0 +1,118 @@
+const expenseService = require('../services/expenseService');
+const userService = require('../services/userService');
+
+const getExpenses = (req, res) => {
+  try {
+    const { userId, categories, from, to } = req.query;
+
+    let expensesResult = expenseService.getAllExpense();
```

Comment:
The function `getAllExpense` should be checked to ensure it returns an array of expenses. If it returns a promise, you need to handle it asynchronously using `await` or `.then()`.